packages/shared/components/profitability/base-profitability.tsx

```diff
@@ -3,29 +3,32 @@ import * as React from "react";
 
 import { PROFITABILITY_VARIANT } from "./profitability.helper";
 
-interface IBaseProfitabilityProps {
+const BaseProfitability: React.FC<Props> = ({
+  className,
+  variant,
+  isPositive,
+  isNegative,
+  children
+}) => {
+  return (
+    <div
+      className={classNames("profitability", className, {
+        "profitability--positive": isPositive,
+        "profitability--negative": isNegative,
+        "profitability--chips": variant === PROFITABILITY_VARIANT.CHIPS
+      })}
+    >
+      {children}
+    </div>
+  );
+};
+
+export default React.memo(BaseProfitability);
```

Comment:
Change to `const BaseProfitability = React.memo(_BaseProfitability); export default BaseProfitability`